src/pages/home-page/hero-image/hero-image.module.scss

```diff
@@ -58,3 +59,50 @@
     align-content: center;
     justify-items: center;
 }
+
+// --- MEDIA QUERIES ---
+
+@media (1400px <=width) {
+    .root {
+        background-color: aqua;
+    }
+
+    .content {
+        max-width: 1400px;
+    }
+}
+
+@media (980px <=width < 1400px) {
+    .root {
+        background-color: red;
+    }
+}
+
+@media (420px <=width < 980px) {
+    .root {
+        background-color: plum;
+    }
+
+     .overlay-text {
```

Comment:
why is it copied here?

Reply:
it will be updated later. removed unnecessary attributes.